modules/imgproc/src/contours.cpp

```diff
@@ -1372,20 +1539,24 @@ icvFindContoursInInterval( const CvArr* src,
     tmp_prev = upper_line;
     for( j = 0; j < img_size.width; )
     {
-        for( ; j < img_size.width && !ICV_IS_COMPONENT_POINT(src_data[j]); j++ )
-            ;
+#if CV_SSE2
```

Comment:
I think, it makes sense to 1) rename haveSSE2 to haveSIMD, because later on someone might want to add NEON, AVX etc. optimizations 2) add haveSIMD parameter in all configurations; the overhead of passing extra parameter is tiny, comparing to the processing time